src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +16,63 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("ERROR: can't add " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Comment.class, id);
+        } catch (Exception e) {
+            throw new DataProcessingException("ERROR: can't get by id " + id, e);
+        }
     }
 
     @Override
     public List<Comment> getAll() {
-        return null;
+        try (Session session = factory.openSession();) {
+            Query<Comment> getAllMoviesQuery = session.createQuery("from Comment", Comment.class);
+            return getAllMoviesQuery.getResultList();
+        } catch (Exception e) {
+            throw new DataProcessingException("ERROR: can't get all", e);
+        }
     }
 
     @Override
     public void remove(Comment entity) {
```

Comment:
```suggestion
    public void remove(Comment comment) {
```
same in other methods